tutorials/classical_simulation/Simulating Sequences with Errors and Noise.ipynb

```diff
@@ -0,0 +1,565 @@
+{
```

Comment:
<p>"We act on this atom with a Constant Pulse, such that it oscillates between the ground and excited Rydberg state (Rabi oscillations):"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>